agent/app/agent_unix_test.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows
+// +build linux
```

Comment:
Makes it stronger. Just fortifying the intent that it will cause failures if not running on linux.